content/intro-to-storybook/vue/en/deploy.md

```diff
@@ -1,99 +1,155 @@
 
-[Create an account on Netlify](https://app.netlify.com/start) and click to “create site”.
+Hooray! We published Storybook with one command, but manually running a command every time we want to get feedback on UI implementation is repetitive. Ideally, we'd publish the latest version of components whenever we push code. We'll need to continuously deploy Storybook.
 
-![Netlify create site](/intro-to-storybook/netlify-create-site.png)
+## Continuous deployment with Chromatic
 
-Next click the GitHub button to connect Netlify to GitHub. This allows it to access our remote taskbox repo.
+Now that our project is hosted in a GitHub repository, we can use a continuous integration(CI) service to deploy our Storybook automatically. [GitHub Actions](https://github.com/features/actions) is a free CI service that's built into GitHub that makes automatic publishing easy.
 
-Now select the taskbox GitHub repo from the list of options.
+### Add a GitHub Action to deploy Storybook
 
-![Netlify connect to repo](/intro-to-storybook/netlify-account-picker.png)
+In the root folder of our project, create a new directory called `.github` then create another `workflows` directory inside of it.
 
-Configure Netlify by highlighting which build command to run in its CI and which directory the static site is outputted in. For branch choose `master`. Directory is `storybook-static`. Build command use `yarn build-storybook`.
+Create a new file called `chromatic.yml` like the one below. Replace to change `project-token` with your project token.
+
+```yaml
+# .github/workflows/chromatic.yml
+# name of our action
+name: 'Chromatic Deployment'
+# the event that will trigger the action
+on: push
+
+# what the action will do
+jobs:
+  test:
+    # the operating system it will run on
+    runs-on: ubuntu-latest
+    # the list of steps that the action will go through
+    steps:
+      - uses: actions/checkout@v1
+      - run: yarn
+      - uses: chromaui/action@v1
+        # options required to the GitHub chromatic action
+        with:
+          # our project token, to see how to obtain it
+          # refer to https://www.learnstorybook.com/intro-to-storybook/vue/en/deploy/
+          projectToken: project-token
+          token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+<div class="aside"><p>For brevity purposes <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets">GitHub secrets</a> weren't mentioned. Secrets are secure environment variables provided by GitHub so that you don't need to hard code the <code>project-token</code>.</p></div>
+
+### Commit the action
+
+In the command line, issue the following command to add the changes that were done:
+
+```bash
+git add .
+```
+
+Then commit them by issuing:
+
+```bash
+git commit -m "GitHub action setup"
+```
+
+Finally push them to the remote repository with:
+
+```bash
+git push origin master
+```
```

Comment:
@andre-brdoch replace master with main in here as well so that it matches the above.